introduction.md

```diff
@@ -6,3 +6,4 @@ Dojo Rules
 * Never bring live blades on the tatami
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.